README.md

```diff
@@ -33,6 +33,11 @@ TCP example
   include_tag_key true
   tag_key 'tag'
 
+  # Optaional tags
```

Comment:
nits `Optional`

Reply:
Oops, fixed it. Thanks for the feedback!